clients/cli/src/utils.rs

```diff
@@ -178,6 +179,15 @@ pub fn edit_file_with_editor(file_location: &str) -> bool {
     }
 }
 
+pub fn metadatas() -> CliResult<Vec<DecryptedFileMetadata>> {
+    match list_metadatas(&config()?) {
+        Ok(metadatas) => Ok(metadatas),
+        Err(err) => match err {
+            UnexpectedError(msg) => Err(err_unexpected!("{}", msg)),
```

Comment:
I think this is likely not the ideal way to handle this.

Reply:
Fundamentally, this is fine, if we get an error this is what we want to return.

you can just `map_err` tho, anytime you do something like: `Ok(metadatas) => Ok(metadatas),` it's a sign there's probably a smaller way to write it.

roughly it'll end up being something like:
```
list_metadatas(&config()?)
    .map_err(|err| err_unexpected!("{}", err))
```